sklearn/cluster/_dbscan.py

```diff
@@ -46,7 +46,7 @@ def dbscan(X, eps=0.5, *, min_samples=5, metric='minkowski',
         The number of samples (or total weight) in a neighborhood for a point
         to be considered as a core point. This includes the point itself.
 
-    metric : string, or callable
+    metric : string or callable
```

Comment:
Same here (`str` instead of `string`). In fact, since the default value is `minkowski`, this information should be provided with `default="minkowski")`.

Reply:
ok